data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/special_calculations.cfg

```diff
@@ -130,23 +130,23 @@
 # API(s) being tested: [attacks]add=,[attacks]sub=
 ##
 # Expected end state:
-# Bob's attack has 2 strikes. When both add and sub are used in the same [attacks] tag, only the sub is used.
+# Bob's attack has 4 strikes. When both add and sub are used in the same [attacks] tag, both add and sub are used.
 #####
-{ONE_CALCULATION_UNIT_TEST add_sub_combined 5 (add,sub=2,3) 2}
+{ONE_CALCULATION_UNIT_TEST add_sub_combined 5 (add,sub=2,3) 4}
 #####
 # API(s) being tested: [attacks]add=,[attacks]sub=
 ##
 # Expected end state:
-# Bob's attack has 2 strikes. Even in separate [attacks] tags, without separate ids the sub overwrites the add.
+# Bob's attack has 4 strikes. In separate [attacks] tags, without separate ids  both add and sub are used.
 #####
-{TWO_CALCULATION_UNIT_TEST add_sub_separated 5 (cumulative,add=yes,2) (cumulative,sub=yes,3) 2}
+{TWO_CALCULATION_UNIT_TEST add_sub_separated 5 (cumulative,add=yes,2) (cumulative,sub=yes,3) 4}
```

Comment:
There's a typo in the already-merged version of this test, and it's testing exactly the same code as `add_sub_cumulative`. This one shouldn't have had the `cumulative=yes` parts, it should have just been (and I've retested and confirm this passes too)
```
{TWO_CALCULATION_UNIT_TEST add_sub_separated 5 (add=2) (sub=3) 2} # result will become 4 with this PR
```

Reply:
fixed

Reply:
Test's going to fail, because the result will become 4 with this PR.

However, I'd like to merge #9313 first, please.

Reply:
ok, i fix and rebase after you.

Reply:
i lready done a fix of test, sorry. when your commit willbe merged, i done another rebase.